kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -66,47 +67,31 @@ public void testGetFigure() {
     }
 
     @Test
-    public void testShowFiguresSquare() {
+    public void testShowFigures() {
         //Given
         Shape addedShape = new Square(4);
-        ArrayList<Shape> testList = new ArrayList<Shape>();
-        String expected = "Square";
-        testList.add(addedShape);
-        //When
-        String actual = testList.get(0).getShapeName();
-        //Then
-        Assert.assertEquals(expected, actual);
-    }
-
-    @Test
-    public void testShowFiguresTriangle() {
-        //Given
-        Shape addedShape = new Triangle(13.0, 13.0, 24.0);
-        ArrayList<Shape> testList = new ArrayList<Shape>();
-        String expected = "Triangle";
-        testList.add(addedShape);
+        Shape addedShape2 = new Triangle(13.0, 13.0);
+        Shape addedShape3 = new Circle(1.0);
+        List<Shape> shapeList = new ArrayList<Shape>();
+        shapeList.add(addedShape);
+        shapeList.add(addedShape2);
+        shapeList.add(addedShape3);
+        List<String> expected = new ArrayList<String>();
+        expected.add("Square");
+        expected.add("Triangle");
+        expected.add("Circle");
         //When
-        String actual = testList.get(0).getShapeName();
-        //Then
-        Assert.assertEquals(expected, actual);
-    }
-
-    @Test
-    public void testShowFiguresCircle() {
-        //Given
-        Shape addedShape = new Circle(1.0);
-        ArrayList<Shape> testList = new ArrayList<Shape>();
-        String expected = "Circle";
-        testList.add(addedShape);
-        //When
-        String actual = testList.get(0).getShapeName();
+        List<String> actual = new ArrayList<String>();
+        for ( Shape shape : shapeList) {
```

Comment:
Zastanawiam się, co ty właściwie chcesz tu testować. Metoda show moim zdaniem powinna wypisywać wszystkie figury w jakiś określony sposób, np. oddzielone średnikiem - tak jak okresliłeś to w metodach toString() dla każdej konkretnej klasy dziedziczącej po Shape. Np. ja u siebie dałbym to w ten sposób: [Triangle(2,4); Circle(5); Square(2)]. Teraz mając te trzy obiekty (triangle, circle, square) buduję w teście jak powinien wyglądać expected string (tak jak powyżej [...]) i porównuję to z wynikiem metody showFigure()